nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -408,11 +417,14 @@ def connect_thread_func(self):
             # The traceback usually contains 3 lines, so collect the first
             # three lines then display it.
             if got_error == 2:
-                ssh_channel.close()
```

Comment:
Instead of just calling `ssh_channel.close()`, this function now calls `self.close` which closes both the `ssh_client`, the `udp_socket` and does the `terminate_client`.